src/index.js

```diff
@@ -104,9 +108,9 @@ var createElement = function(node, lifecycle, eventProxy, isSvg) {
       : document.createElement(node.name)
 
   var props = node.props
-  if (props.onCreate) {
+  if (props.oncreate) {
```

Comment:
Thank you for this :clap:

Reply:
Hah 😅 unintentional side effect of trying to keep things as uniform/simple as possible.

Reply:
if remove this line at all, it will be possible to use both `onCreate` and `oncreate` similar to all other events like `onClick` and `onclick`.